File: duolingo_rum/Models/Topic.cs
using System;
using System.Collections.Generic;

namespace duolingo_rum.Models;

public partial class Topic
{
    public int Id { get; set; }

    public int LanguageId { get; set; }

    public string Name { get; set; } = null!;

    public string? Description { get; set; }

    public string? IconEmoji { get; set; }

    public int? SortOrder { get; set; }

    public virtual Language Language { get; set; } = null!;

    public virtual ICollection<Word> Words { get; set; } = new List<Word>();
}

File: android-app/app/src/main/java/com/linguaai/app/models/Topic.kt
package com.linguaai.app.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Topic(
    val id: Int = 0,
    @SerialName("language_id")
    val languageId: Int = 0,
    val name: String = "",
    val description: String? = null,
    @SerialName("icon_emoji")
    val iconEmoji: String? = null,
    @SerialName("sort_order")
    val sortOrder: Int? = 0
)
